native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/AppDelegate.m

```diff
@@ -66,14 +66,12 @@ - (id)init
         [SFSDKDatasharingHelper sharedInstance].appGroupName = config.appGroupName;
         [SFSDKDatasharingHelper sharedInstance].appGroupEnabled = config.appGroupsEnabled;
         
-        // Need to use SmartSyncSDKManager when using SmartSync
-        [SalesforceSDKManager setInstanceClass:[SmartSyncSDKManager class]];
-        [SmartSyncSDKManager sharedManager].appConfig.remoteAccessConsumerKey = config.remoteAccessConsumerKey;
-        [SmartSyncSDKManager sharedManager].appConfig.oauthRedirectURI = config.oauthRedirectURI;
-        [SmartSyncSDKManager sharedManager].appConfig.oauthScopes = [NSSet setWithArray:config.oauthScopes];
-        __weak typeof(self) weakSelf = self;
-        [[SmartSyncSDKManager sharedManager] addDelegate:self];
+        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleUserDidLogout:) name:kSFNotificationUserDidLogout object:nil];
         
+        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleUserDidSwitch:) name:kSFNotificationUserDidSwitch  object:nil];
+        [SmartSyncSDKManager initializeSDK];
+        [[SmartSyncSDKManager sharedManager] addDelegate:self];
```

Comment:
Why do we still need to be a delegate of the SDK manager? Couldn't we do everything with notification?

Reply:
@wmathurin the delegate was already  part of  the sample to clear keyboard on backgrounding

Reply:
Ah ok.